src/mkdocstrings/templates/python/material/class.html

```diff
@@ -22,7 +22,14 @@
           class="doc doc-heading",
           toc_label=class.name) %}
 
-        <code>{% if show_full_path %}{{ class.path }}{% else %}{{ class.name }}{% endif %}</code>
+        <code>
+          {% if show_full_path %}{{ class.path }}{% else %}{{ class.name }}{% endif %}
+          {% if config.show_bases %}
+            ({% for base in class.bases -%}
```

Comment:
One more thing: do we want to render `()` when there aren't any base class? I would say no. In that case an additional condition on `class.bases`  would be used before the loop.

Reply:
Maybe you can combine both conditions like so:

```
{% if config.show_bases and class.bases %}
```

Reply:
A Python 3 class with no explicit bases implicitly gets `object` as a base, so this can't happen.  We could omit the list entirely if `object` is the only base.

Reply:
Ah, true, my bad. Then yeah, we must decide if showing `(object)` has value or not. I'd say it doesn't, what do you think?

Reply:
The official python 3 docs seem to omit it in code and use the implicit behavior, so I changed it to match.